testsuite/MDAnalysisTests/core/test_atom.py

```diff
@@ -0,0 +1,150 @@
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+import numpy as np
+from numpy.testing import (
+    dec,
+    raises,
+    assert_almost_equal,
+    assert_equal,
+    assert_raises,
+)
+
+import MDAnalysis as mda
+from MDAnalysis import NoDataError
+
+from MDAnalysisTests.datafiles import (
+    PSF, DCD,
+    XYZ_mini,
+)
+from MDAnalysisTests import parser_not_found
+
+
+class TestAtom(object):
+    # Legacy tests from before 363
+    """Tests of Atom."""
+
+    @dec.skipif(parser_not_found('DCD'),
+                'DCD parser not available. Are you using python 3?')
+    def setUp(self):
+        """Set up the standard AdK system in implicit solvent."""
+        self.universe = mda.Universe(PSF, DCD)
+        self.atom = self.universe.atoms[1000]  # Leu67:CG
+        self.known_pos = np.array([3.94543672, -12.4060812, -7.26820087],
+                                  dtype=np.float32)
+
+    def tearDown(self):
+        del self.universe
+        del self.atom
+        del self.known_pos
+
+    def test_attributes_names(self):
+        a = self.atom
+        assert_equal(a.name, 'CG')
+        assert_equal(a.resname, 'LEU')
+
+    def test_setting_attribute_name(self):
+        self.atom.name = 'AA'
+        assert_equal(self.atom.name, 'AA')
+
+    def test_setting_attribute_type(self):
+        self.atom.type = 'Z'
+        assert_equal(self.atom.type, 'Z')
+
+    def test_setting_attribute_mass(self):
+        self.atom.mass = 13
+        assert_equal(self.atom.mass, 13)
+
+    def test_setting_attributes_charge(self):
+        self.atom.charge = 6
+        assert_equal(self.atom.charge, 6)
+
+    def test_attributes_positions(self):
+        a = self.atom
+        # new position property (mutable)
+        assert_almost_equal(a.position, self.known_pos)
+        pos = a.position + 3.14
+        a.position = pos
+        assert_almost_equal(a.position, pos)
+
+    def test_atom_selection(self):
+        asel = self.universe.select_atoms('atom 4AKE 67 CG').atoms[0]
+        assert_equal(self.atom, asel)
+
+    def test_hierarchy(self):
+        u = self.universe
+        a = self.atom
+        assert_equal(a.segment, u.s4AKE)
+        assert_equal(a.residue, u.residues[66])
+
+    def test_bad_add(self):
+        def bad_add():
+            return self.atom + 1
+
+        assert_raises(TypeError, bad_add)
+
+    def test_add_AG(self):
+        ag = self.universe.atoms[:2]
+
+        ag2 = self.atom + ag
+
+        for at in [self.atom, ag[0], ag[1]]:
+            assert_equal(at in ag2, True)
+
+    def test_no_velo(self):
```

Comment:
https://github.com/MDAnalysis/mdanalysis/blob/develop/testsuite/MDAnalysisTests/core/test_group_traj_access.py#L247

It's very redundant because we systematically go through all possible combinations there.  But they've got some values as these use files, so are more "real"?